action.yml

```diff
@@ -21,6 +29,8 @@ runs:
     - ${{ inputs.command }}
     - "-f"
     - ${{ inputs.filename }}
+    - ${{ inputs.with_url }}
+    - ${{ inputs.with_create }}
```

Comment:
Paired with comments/changes at https://github.com/ketch-apps/alchemer/pull/18/files#r570692126

Not sure this will work because `ketch validate` will have empty strings for these 2 args